packages/devtools_app/lib/src/landing_screen.dart

```diff
@@ -154,12 +154,14 @@ class _LandingScreenBodyState extends State<LandingScreenBody> {
             ],
           ),
         ),
-        const Padding(
-          padding: EdgeInsets.only(left: 20.0),
-        ),
-        ElevatedButton(
-          child: const Text('Connect'),
-          onPressed: connectDebounce.invoke,
+        const SizedBox(width: defaultSpacing),
+        Padding(
+          // Padding to center the connect button with the `TextField`.
```

Comment:
if the font size is different will we lose this centering?
Can you use mainAxisAlignment instead to make things look good? or maybe crossAxisAlignment? I'd have to use the layout explorer to remember which.

Reply:
rearranged some of these widgets so we could use axis alignment instead.